third_party/libgt2/libgt2_engine.bzl

```diff
@@ -0,0 +1,178 @@
+#
+# Copyright (c) 2025 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
+
+def _is_windows(ctx):
+    return ctx.os.name.lower().find("windows") != -1
+
+def libgt2_engine():
+    libgt2_repository(name="_libgt2_engine")
+    new_git_repository(
+        name = "libgt2_engine",
+        remote = "https://github.com/libgit2/libgit2.git",
+        commit = "338e6fb681369ff0537719095e22ce9dc602dbf0", # Dec 28, 2024 - v1.9.0
+        build_file = "@_libgt2_engine//:BUILD",
+    )
+
+    #native.new_local_repository(
+    #    name = "libgt2_engine",
+    #    path = "/libgt2_engine",
+    #    build_file = "@_libgt2_engine//:BUILD",
+    #)
+
+def _impl(repository_ctx):
+    http_proxy = repository_ctx.os.environ.get("http_proxy", "")
+    if http_proxy == "":
+        http_proxy = repository_ctx.os.environ.get("HTTP_PROXY", "")
+    
+    https_proxy = repository_ctx.os.environ.get("https_proxy", "")
+    if https_proxy == "":
+        https_proxy = repository_ctx.os.environ.get("HTTPS_PROXY", "")
+
+    # Create the Python script dynamically
+    repository_ctx.file("remove_japanese_txt.py", """
+import os
+import fnmatch
+import sys
+
+def remove_japanese_txt(directory):
```

Comment:
We are using this in other repo's already. Calling this should be some kind of bazel procedure loaded from *.bzl file

Reply:
Need to placed exactly in the repository ctx directory on every build. This script will remove a directory not only files because we cannot filter the files by extension like it was done in the other python script.